### DISCUSSION TITLE
Anyway to stop d or y copying content to clipboard?

### DISCUSSION THREAD

**Discussion by Dennis-Zhang-SH** [ayamir/nvimdots#797] [Q&A] [answered]
I actually don't want this funciton.

**Comment by ayamir** [comment]
Do you mean the system clipboard?

**Reply by Dennis-Zhang-SH** [reply]
> Do you mean the system clipboard?

Sure.

**Comment by ayamir** [comment]
Comment these lines:
https://github.com/ayamir/nvimdots/blob/38cb531183c6605ec58b21fd84bde18327b1f428/lua/core/options.lua#L19

https://github.com/ayamir/nvimdots/blob/main/lua/modules/plugins/tool.lua#L25-L29

**Reply by Dennis-Zhang-SH** [reply]
> Comment these lines:
> 
> https://github.com/ayamir/nvimdots/blob/38cb531183c6605ec58b21fd84bde18327b1f428/lua/core/options.lua#L19
> 
> https://github.com/ayamir/nvimdots/blob/main/lua/modules/plugins/tool.lua#L25-L29

Still return E475: Invalid value for argument cmd: 'win32yank.exe' is not executable, please tell me how to skip this.

**Reply by ayamir** [reply, marked as answer]
comment this line:
https://github.com/ayamir/nvimdots/blob/38cb531183c6605ec58b21fd84bde18327b1f428/lua/core/init.lua#L152
